addons/pad/py_etherpad/__init__.py

```diff
@@ -186,6 +191,15 @@ def setText(self, padID, text):
             "text": text
         })
 
+    def setHtmlFallbackText(self, padID, html):
+        try:
+            # Prevents malformed HTML errors
+            html_wellformed = '<html>' + html + '</html>'
```

Comment:
If the param html doesn't contains the tags <body></body> etherpad will not return an error but will also not process the html sent.

e.g:
Html sent:
```html
<html><b>test</b></html>
```
Html read:
```html
<!DOCTYPE HTML><html><body><br></body></html>
```